code/modules/power/tesla/energy_ball.dm

```diff
@@ -40,15 +40,16 @@ var/list/blacklisted_tesla_types = list(/obj/machinery/atmospherics,
 	if(!is_orbiting)
 		handle_energy()
 		var/amount_to_move = 2 + orbiting_balls.len * 2
+		var/what_does_the_scouter_say_about_the_balls_power_level = (TESLA_DEFAULT_POWER + (TESLA_MINI_POWER * orbiting_balls.len))
```

Comment:
You may...want to nerf this a bit.

Without even upgrading the coils, you can reach legitimately *insane* power levels; by 30-40 minutes in, again, without upgrades, it'll be pulsing the grid for over 100 million watts.